Sming/Components/spiffs/spiffs_sming.h

```diff
@@ -18,8 +18,8 @@ extern "C" {
 
 #define LOG_PAGE_SIZE       256
 
-void spiffs_mount();
-void spiffs_mount_manual(u32_t phys_addr, u32_t phys_size);
+bool spiffs_mount();
+bool spiffs_mount_manual(u32_t phys_addr, u32_t phys_size);
 void spiffs_unmount();
 bool spiffs_format();
 bool spiffs_format_internal(spiffs_config *cfg);
```

Comment:
I do wonder why `spiffs_format_internal` is defined in the header. If it's intended for application use then I'd expect it to be called `spiffs_format_manual(u32_t, u32_t)`. Could we remove it, I wonder?

Reply:
I agree with you, and since the application can pass configuration with arbitrary block size and page size, this could lead to some issue